hack/jenkins/e2e.sh

```diff
@@ -49,29 +53,48 @@ echo "E2E_OPT: ${E2E_OPT}"                         # hack/e2e.go options
 echo "E2E_SET_CLUSTER_API_VERSION: ${E2E_SET_CLUSTER_API_VERSION:-<not set>}" # optional, for GKE, set CLUSTER_API_VERSION to git hash
 echo "--------------------------------------------------------------------------------"
 
-# GCE variables
-export INSTANCE_PREFIX=${E2E_CLUSTER_NAME}
-export KUBE_GCE_ZONE=${E2E_ZONE}
-export KUBE_GCE_NETWORK=${E2E_NETWORK}
 
-# GKE variables
-export CLUSTER_NAME=${E2E_CLUSTER_NAME}
-export ZONE=${E2E_ZONE}
-export KUBE_GKE_NETWORK=${E2E_NETWORK}
+if [[ "${KUBERNETES_PROVIDER}" == "aws" ]]; then
+    export KUBE_AWS_INSTANCE_PREFIX=${E2E_CLUSTER_NAME}
+    export KUBE_AWS_ZONE=${E2E_ZONE}
+else
+    # GCE variables
+    export INSTANCE_PREFIX=${E2E_CLUSTER_NAME}
+    export KUBE_GCE_ZONE=${E2E_ZONE}
+    export KUBE_GCE_NETWORK=${E2E_NETWORK}
+
+    # GKE variables
+    export CLUSTER_NAME=${E2E_CLUSTER_NAME}
+    export ZONE=${E2E_ZONE}
+    export KUBE_GKE_NETWORK=${E2E_NETWORK}
+fi
+
 
 export PATH=${PATH}:/usr/local/go/bin
-export HOME=${WORKSPACE} # Nothing should want Jenkins $HOME
 export KUBE_SKIP_CONFIRMATIONS=y
 
-# sudo gcloud components update -q
+if [[ -d _output/ ]]; then
+    echo "Found _output/ directory; will use binaries from there"
+    cp _output/release-tars/kubernetes*.tar.gz .
+else
+    echo "Pulling binaries from GCS"
+    if [[ $(find . | wc -l) != 1 ]]; then
+        echo $PWD not empty, bailing!
+        exit 1
+    fi
+
+    # sudo gcloud components update -q
```

Comment:
@zmerlynn Do we do this elsewhere in our jenkins? It doesn't seem like this should be commented out.